src/Core/EventMap.cs

```diff
@@ -1,10 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
+﻿// <copyright file="EventMap.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
 
 namespace CR.AggregateRepository.Core
 {
+    using System;
+    using System.Collections.Generic;
+
+    /// <inheritdoc />
+    /// <summary>
+    /// Matches a type to an action.
```

Comment:
We should probably explain what the EventMap is actually for in the code, it is always just referred to as an `EventMap` but we never detail how it is used.